src/pubsub/message/topic-descriptor.proto

```diff
@@ -1,10 +1,10 @@
-'use strict'
-module.exports = `
+syntax = "proto3";
+
 // topicCID = cid(merkledag_protobuf(topicDescriptor)); (not the topic.name)
 message TopicDescriptor {
   optional string name = 1;
   optional AuthOpts auth = 2;
-  optional EncOpts enc = 2;
+  optional EncOpts enc = 3;
```

Comment:
Oops.

Reply:
yes, this was awkward! Go and the spec were right, but I guess there is no one using topic descriptors in JS so far
Spec: https://github.com/libp2p/specs/tree/master/pubsub#the-topic-descriptor
Go: https://github.com/libp2p/go-libp2p-pubsub/blob/master/pb/rpc.proto#L61

Reply:
Types catching all the bugs 🎉